### PR TITLE
Change Compact API names with universal ones.

### PR DESCRIPTION
**Pull request by sancar**
These names are chosen as they are not java centric and more polyglot.

### CODE REVIEW

**Comment by sancar** [comment]
run-lab-run

**Comment by sancar** [comment]
run-lab-run

**Comment by sancar** [comment]
run-lab-run

**Review by mdumandag — APPROVED** [review]

File: hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java [hunk 1]
```diff
@@ -869,251 +869,251 @@ public void writeJsonFormattedField(StringBuilder stringBuilder, AbstractGeneric
...
             @Override
             public int hashCode(GenericRecord record, String fieldName) {
-                return Arrays.hashCode(record.getArrayOfNullableDoubles(fieldName));
+                return Arrays.hashCode(record.getArrayOfNullableFloat64s(fieldName));
             }
 
             @Override
             public void writeFieldFromRecordToWriter(DefaultCompactWriter writer, GenericRecord record, String fieldName) {
-                writer.writeArrayOfNullableDoubles(fieldName, record.getArrayOfNullableDoubles(fieldName));
+                writer.writeArrayOfNullableFloat64s(fieldName, record.getArrayOfNullableFloat64s(fieldName));
```
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java:1116]: I don't think we need the `s` at the end. I would prefer `writeArrayOfNullableFloat64`

File: hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/DefaultCompactWriter.java [hunk 2]
```diff
@@ -304,28 +304,28 @@ public void writeArrayOfBooleans(@Nonnull String fieldName, @Nullable boolean[]
     }
 
     @Override
-    public void writeArrayOfInts(@Nonnull String fieldName, @Nullable int[] values) {
-        writeVariableSizeField(fieldName, ARRAY_OF_INTS, values, ObjectDataOutput::writeIntArray);
+    public void writeArrayOfInt32s(@Nonnull String fieldName, @Nullable int[] values) {
+        writeVariableSizeField(fieldName, ARRAY_OF_INT32S, values, ObjectDataOutput::writeIntArray);
```
Comment (ihsandemir) [inline hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/DefaultCompactWriter.java:308]: No `S` at the end.

File: hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java (hunk 1 above)
Comment (sancar) [reply hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java]: @ihsandemir updated the pr. Can you check again ?

**Review by ihsandemir — APPROVED** [review]

**Pull request merged**